src/Security/Security/ChangeLog.md

```diff
@@ -19,6 +19,15 @@
 -->
 ## Upcoming Release
 
+## Version 0.8.0
```

Comment:
Please remove the "## Version 0.8.0" and move change log under "## Upcoming Release"